.travis.yml

```diff
@@ -247,10 +247,7 @@ before_install:
   - if [ "$TRAVIS_OS_NAME" = "windows" ]; then cinst StrawberryPerl $([ "$PLATFORM" = "x86" ] && echo --forcex86) --version "$TRAVIS_PERL_VERSION"; export PATH="/c/Strawberry/perl/site/bin:/c/Strawberry/perl/bin:/c/Strawberry/c/bin:$PATH"; perl -MConfig -e 'print "#!/bin/sh\n$Config{make} \"\$@\"\n"' > /c/Strawberry/perl/site/bin/make; chmod +x /c/Strawberry/perl/site/bin/make; fi
 
 install:
-  - perl Makefile.PL
-  - make regen
-# ensure allfiles are git up-to-date [fail if make regen alters some files]
-  - git diff --quiet
+  - true
```

Comment:
why removing this rule? if the test files are ignored `git diff --quiet` will come clean
either we should leave it as it, or remove the whole `install` section?

Reply:
`make regen` + `git diff --quiet` was there to check that all autogenerated files are up-to-date. As this pull request removes those autogenerated files from tree, there is no need to check for `make regen` + `git diff --quiet`.

`perl Makefile.PL` was there to generate Makefile, so `make regen` could be used.

Removing whole install section is not possible as in this case Travis would use default install section which calls some perl scripts which needs Perl 5.8. And because Devel::PPPort is testing on Travis also older Perl versions, it is not possible to use default install section.

Therefore it is needed to define some install section, but currently we do not need to execute any command in it anymore.